internal/server/filesystem/server.go

```diff
@@ -2,24 +2,227 @@ package filesystem
 
 import (
 	"context"
+	"fmt"
+	"regexp"
+	"strings"
 
-	"github.com/kubernetes-csi/csi-proxy/client/api"
 	"github.com/kubernetes-csi/csi-proxy/client/apiversion"
 	"github.com/kubernetes-csi/csi-proxy/internal/server/filesystem/internal"
 )
 
-// Server is the internal server for the filesytem API group.
-type Server struct{}
+type Server struct {
+	kubeletCSIPluginsPath string
+	kubeletPodPath        string
+	hostAPI               API
+}
+
+const MAX_PATH_LENGTH_WINDOWS = 260
```

Comment:
Nit: this will certainly be used in many other places in this repo, might be worth putting it into an `internal/utils.go` file or similar?